open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/README.md

```diff
@@ -7,6 +7,78 @@
 
 The Open Lineage Janus connector allows the Open Lineages Services to connect with a JanusGraph database.
 
+The Connector can be configured to connect to either an embedded JanusGraph database or a standalone JanusGraph server.
+   
+Configuring the properties of the JanusGraph client when connecting to a server should be done following the [official documentation](https://docs.janusgraph.org/basics/configuration/)
```

Comment:
I would suggest to re phase this part because it already suggests client-server mode and this is confusing when we have embedded janus. Suggest to remove "when connecting to a server".